### ISSUE TITLE
After clicking the name...

### ISSUE THREAD

**Issue by onimaca** [Neo11/ChitChat#13]
I think it would be great if after clicking the name of a contact the cursor would instantly move to reply field

**Comment by c-o-m-m-a-n-d-e-r** [comment]
yep, this is really annoing ... think that can we solve

**Comment by c-o-m-m-a-n-d-e-r** [comment]
Build this function and already merged into the Base...
will be in the next Build @Neo11 have to release :)

**Comment by onimaca** [comment]
It still doesnt work is i search the name of someone and click the name after

**Comment by c-o-m-m-a-n-d-e-r** [comment]
ok but when you click on a contact where chat exists it works ?!
I think i know the Problem ... will fix that

**Comment by onimaca** [comment]
it happens always i use the searchbox

**Comment by c-o-m-m-a-n-d-e-r** [comment]
yes, searchbox i know but when you click direct on a chat contact name ?!
without using searchbox...

**Comment by onimaca** [comment]
if i dont use the searchbox it works well

**Comment by c-o-m-m-a-n-d-e-r** [comment]
ok thanks ... i will look after it ...
never used searchbox before ;) so "contact" was for me a ready chat hehe ...

**Comment by c-o-m-m-a-n-d-e-r** [comment]
I think this is working now isn't it?
